include/dpp/discordvoiceclient.h

```diff
@@ -38,6 +38,7 @@
 #include <dpp/dispatcher.h>
 #include <dpp/cluster.h>
 #include <dpp/discordevents.h>
+#include <dpp/isa_detection.h>
```

Comment:
we cannot include this here.
including this here means that the end user code must set AVX_TYPE. it cannot be included in a header.